### ISSUE TITLE
Prepare 2 slice in one UE deployment

### ISSUE THREAD

**Issue by pirog-spb** [edgecomllc/eupf#420]
Add new scenario for comparing eupf and upf from open5gs using one UE connected to two separate slices. Each slice contains own upf.

Accepatance criteria:

- [x] Prepare deployment with UERANSIM UE and 2 slices
- [x] Present to team possibility to test several upfs via one window

**Comment by PapaySail** [comment]
![image](https://github.com/edgecomllc/eupf/assets/125686203/2f3fdc10-8d7b-411c-9f51-bba6a3228080)


<details><summary>Details</summary>
<p>

OK EU log:
```

``` 


</p>
</details>

**Comment by PapaySail** [comment]
<details><summary>OK two PDU sessions throught different slices</summary>
<p>

```
GNB_IP: 10.233.64.23
Launching ue: nr-ue -c ue.yaml
UERANSIM v3.2.6
[2023-10-04 07:03:45.572] [nas] [info] UE switches to state [MM-DEREGISTERED/PLMN-SEARCH]
[2023-10-04 07:03:45.573] [rrc] [debug] New signal detected for cell[1], total [1] cells in coverage
[2023-10-04 07:03:45.573] [nas] [info] Selected plmn[999/70]
[2023-10-04 07:03:45.573] [rrc] [info] Selected cell plmn[999/70] tac[1] category[SUITABLE]
[2023-10-04 07:03:45.573] [nas] [info] UE switches to state [MM-DEREGISTERED/PS]
[2023-10-04 07:03:45.573] [nas] [info] UE switches to state [MM-DEREGISTERED/NORMAL-SERVICE]
[2023-10-04 07:03:45.573] [nas] [debug] Initial registration required due to [MM-DEREG-NORMAL-SERVICE]
[2023-10-04 07:03:45.573] [nas] [debug] UAC access attempt is allowed for identity[0], category[MO_sig]
[2023-10-04 07:03:45.573] [nas] [debug] Sending Initial Registration
[2023-10-04 07:03:45.574] [nas] [info] UE switches to state [MM-REGISTER-INITIATED]
[2023-10-04 07:03:45.574] [rrc] [debug] Sending RRC Setup Request
[2023-10-04 07:03:45.574] [rrc] [info] RRC connection established
[2023-10-04 07:03:45.574] [rrc] [info] UE switches to state [RRC-CONNECTED]
[2023-10-04 07:03:45.574] [nas] [info] UE switches to state [CM-CONNECTED]
[2023-10-04 07:03:45.582] [nas] [debug] Authentication Request received
[2023-10-04 07:03:45.587] [nas] [debug] Security Mode Command received
[2023-10-04 07:03:45.587] [nas] [debug] Selected integrity[2] ciphering[0]
[2023-10-04 07:03:45.600] [nas] [debug] Registration accept received
[2023-10-04 07:03:45.600] [nas] [info] UE switches to state [MM-REGISTERED/NORMAL-SERVICE]
[2023-10-04 07:03:45.600] [nas] [debug] Sending Registration Complete
[2023-10-04 07:03:45.600] [nas] [info] Initial Registration is successful
[2023-10-04 07:03:45.600] [nas] [debug] Sending PDU Session Establishment Request
[2023-10-04 07:03:45.600] [nas] [debug] UAC access attempt is allowed for identity[0], category[MO_sig]
[2023-10-04 07:03:45.600] [nas] [debug] Sending PDU Session Establishment Request
[2023-10-04 07:03:45.600] [nas] [debug] UAC access attempt is allowed for identity[0], category[MO_sig]
[2023-10-04 07:03:45.804] [nas] [debug] Configuration Update Command received
[2023-10-04 07:03:45.837] [nas] [debug] PDU Session Establishment Accept received
[2023-10-04 07:03:45.837] [nas] [info] PDU Session establishment is successful PSI[2]
[2023-10-04 07:03:45.850] [nas] [debug] PDU Session Establishment Accept received
[2023-10-04 07:03:45.851] [nas] [info] PDU Session establishment is successful PSI[1]
[2023-10-04 07:03:45.855] [app] [info] Connection setup for PDU session[2] is successful, TUN interface[uesimtun0, 10.45.0.2] is up.
[2023-10-04 07:03:45.882] [app] [info] Connection setup for PDU session[1] is successful, TUN interface[uesimtun1, 10.46.0.2] is up.
Stream closed EOF for open5gslice/ueransim-ues2-77647d77f5-596c8 (ues)

``` 

</p>
</details>
